changelog.d/20211213_115038_yadudoc1729_add_payload_storage.md

```diff
@@ -0,0 +1,13 @@
+### Added
+
+- `funcx_common.task_storage.TaskStorage` and it's child classes (`ImplicitRedisStorage` and
+    `RedisS3Storage`) now supports `store_payload` and `get_payload` methods.
+- `Kind` enum to represent `result` and `payload` types.
+
+
+### Changed
+
+- `funcx_common.task_storage.RedisS3Storage`'s `_store_to_s3` and `_get_from_s3` method
```

Comment:
As these are internal methods, we don't shouldn't have them in the changelog for library consumers.
For the outside world, it makes the changelog noisier without adding enough value.

I feel similarly about mention of `Kind` above.

Reply:
Removed